### ISSUE TITLE
Adjustable vocab reading priority

### ISSUE THREAD

**Issue by stbotman** [SYtor/Kanji-Dojo#182]
Both in existing vocab lists and when adding expression into custom one, there is a chance that instead of kanji reading hiragana will be added. I would argue that since kanji has furigana it's superior as default. I understand that this could be not so straightforward for any random word, but at least when adding word using plus button I want to get what I see. I hope something could be done.

PS Thank you for your hard work, this app is the best one of this kind I tried hands down.

**Comment by syt0r** [comment]
Yes, default priorities can get a bit random for some words so it's hard to do something without manually editing everything. It's quite complicated to implement customizable sorting right now but this is certainly a very useful feature
